docs/tools/kyuubi-adminctl.md

```diff
@@ -0,0 +1,55 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# Administer kyuubi server Tool
+
```

Comment:
please describe what this tool is

Reply:
as I don't understand why refreshing config is admin behavior but creating/deleting a server is not.

Reply:
can we move the server actions into kyuubi-adminctl later?

Reply:
1. kyuubi-admin looks better
2. > can we move the server actions into kyuubi-adminctl later?

we can but better duplicate the functions and deprecate them in ctl first

Reply:
> 1. kyuubi-admin looks better
It sounds good and does not conflicts with kyuubi-ctl.

We can refactor the kyuubi-ctl in long term.